CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Add AddSentryTag and AddSentryContext Extensions for exception class (#834) @lucas-zimerman
 
+### Changes
```

Comment:
It's a feature to folks outside ASP.NET Core :)

And to ASP.NET Core 2.x users.

```suggestion
### Features
```